doc/contributing/coding-conventions.xml

```diff
@@ -652,6 +662,17 @@ args.stdenv.mkDerivation (args // {
          </variablelist>
         </listitem>
        </varlistentry>
+       <varlistentry>
+        <term>
+         If it’s a <emphasis>theme</emphasis> for a <emphasis>desktop environment</emphasis>,
+         a <emphasis>window manager</emphasis> or a <emphasis>display manager</emphasis>:
```

Comment:
I guess this means we won't regroup sound themes here?
Like https://github.com/NixOS/nixpkgs/tree/master/pkgs/data/misc/sound-theme-freedesktop.

Reply:
(saw this after)
no my thinking is that sound themes would under their own arbo, just like icons